docs/setup/independent/create-cluster-kubeadm.md

```diff
@@ -174,6 +174,26 @@ token can add authenticated nodes to your cluster.  These tokens can be listed,
 created and deleted with the `kubeadm token` command.  See the [reference
 guide](/docs/admin/kubeadm/#manage-tokens).
 
+#### Configure Kubectl
+
+`kubeadm init` generates several configuration files in directory /etc/kubernetes. 
+Kubectl can use them to talk with the cluster. For example, to use the admin.conf, 
+run:
+```
+mkdir -p ~/.kube
+sudo cp /etc/kubernetes/admin.conf ~/.kube/config
+sudo chmod +r ~/.kube/config
```

Comment:
This looks very unsafe. You probably want to chown it to your user and chmod it to 600.